docs/content/integrations/airflow.mdx

```diff
@@ -123,6 +123,35 @@ The `image` argument is the name of the Docker image. Running in a containerized
 
 If you want your containerized job to be available to Airflow operators running on other machines (for example, in environments where Airflow workers are running remotely) you'll need to push your Docker image to a Docker registry so that remote instances of Docker can pull the image by name, or otherwise ensure that the image is available on remote nodes.
 
+## Ingesting Airflow operators
+
+Dagster can convert Airflow operators to Dagster ops, for any Airflow operators that require no instance-wide configuration via `airflow.cfg`. Using <PyObject module="dagster_airflow" object="airflow_operator_to_op"/>, you can convert instantiated Airflow operators to be executed within a Dagster op. Optionally, you can pass in a list of Airflow connection objects utilized by the operator.
+
+```python file=../../airflow_ingest/airflow_operator_to_op.py startafter=start_operator_to_op_1 endbefore=end_operator_to_op_1
+http_task = SimpleHttpOperator(task_id="http_task", method="GET", endpoint="images")
+connections = [Connection(conn_id="http_default", host="https://google.com")]
+dagster_op = airflow_operator_to_op(http_task, connections=connections)
+
+
+@job
+def my_http_job():
+    dagster_op()
+```
+
+In order to specify extra parameters to an instantiated Airflow connection, pass a JSON string into the `set_extra()` function of the Airflow connection object.
```

Comment:
```suggestion
To specify extra parameters to an instantiated Airflow connection, pass a JSON string into the `set_extra()` function of the Airflow connection object:
```

Other small things - suggesting we drop `In order` at the start to shorten the sentence, and add a colon at the end to tie it to the example.